controllers/chat/getSignedChannelList.js

```diff
@@ -49,7 +71,7 @@ const getSignedChannelList = async (req, res) => {
 
       const batchSize = Math.min(30, limit - totalFetched);
       promisedChannels.push(
-        __queryChannelBuilder(userId, batchSize, Number(totalFetched) + Number(offset))
+        __queryChannelBuilder(client, userId, batchSize, Number(totalFetched) + Number(offset))
       );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here is consistent with the modification made at the beginning of the file where `__queryChannelBuilder` now takes an additional parameter `client`. This change is fine as long as the usage of `__queryChannelBuilder` throughout the codebase is updated accordingly.



<!-- This is an auto-generated comment by OSS CodeRabbit -->